02-oop-inheritance/js/script.js

```diff
@@ -0,0 +1,58 @@
+var Movie = function (title,year){
+    this.title = title;
+    this.year = year;
+    this.observerList = [];
+
+    this.get = function (attribute){
+        return this[attribute];
+    };
+    this.set = function (attribute, value){
+      this[attribute]= value;
+    };
+    this.startplaying = function (){
+        for( i = 0; i < observerList.Count; i++) {
+            observerList[i].notifyplaying(this);
+        };
+    };
+    this.stopPlaying = function(){
+        for( i = 0; i < observerList.Count; i++) {
+            observerList[i].notifyStop(this);
+        };
+    };
+    this.subscribeObserver = function(observer){
+        this.observerList.push(observer);
```

Comment:
This should be defined here? What if I had multiple classes that should be observed?